apps/zoomcallstatus/zoom_call_status.star

```diff
@@ -0,0 +1,105 @@
+""")
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "email",
+                name = "Zoom User Email",
+                desc = "Your personal zoom email.",
+                icon = "user",
+            ),
+            schema.Text(
+                id = "token",
+                name = "Zoom JWT Token",
```

Comment:
Looks like Zoom have support for oauth2 as well. It's a bit tricky to get it working, but thought I should mention it in case you hadn't seen it. =)